MarkBind/markbind
# Install Bootstrap-Vue

**What is the purpose of this pull request? (put "X" next to an item, remove the rest)**

• [X] New feature

~~WIP: Not all components uses Bootstrap 4 yet, need to wait them to be migrated over first.~~

~~This PR provides the developers an opportunity to test their components for compatibility with Bootstrap v4.~~

This PR provides the foundation of migrating MarkBind to Bootstrap 4. After the components are ported over in `vue-strap`, the new `vue-strap` will be installed and the migration would be considered complete.






**What is the rationale for this request?**
User would want some new components available in Bootstrap-vue.

**What changes did you make? (Give an overview)**
1. Installed Bootstrap v4.
2. Installed Bootstrap-Vue v2.

**Provide some example code that this change will affect:**
NIL

**Is there anything you'd like reviewers to focus on?**
NIL

**Testing instructions:**
1. Run `markbind serve docs`.
   a. Look at the "Components Reference" page and ensure that everything looks fine.
   b. Look at the general page and ensure that everything looks fine.

**Comparison of bootstrap 3 and bootstrap 4 (With migration):**

Bootstrap 3 | Bootstrap 4
-- | --
<img src="https://user-images.githubusercontent.com/3168908/42140564-be65c79c-7dd3-11e8-9ac1-5638e7bf9b4d.png" width="300" /> | <img src="https://user-images.githubusercontent.com/3168908/42069537-f246d1d8-7b84-11e8-9b94-6914aa740150.png" width="300" />

Migration Documentation / Guides |
--- |
[Official guide and change log](http://getbootstrap.com/docs/4.0/migration/) 
[Migrating Glyphicons](https://stackoverflow.com/questions/32612690/bootstrap-4-glyphicons-migration)
